plugins/obs-webrtc/whip-output.cpp

```diff
@@ -368,6 +424,24 @@ bool WHIPOutput::Connect()
 	CURLU *url_builder = curl_url();
 	auto last_location_header = location_headers.back();
 
+	struct curl_header *prev = nullptr;
+	struct curl_header *h = nullptr;
+	std::vector<rtc::IceServer> iceServers;
+	while ((h = curl_easy_nextheader(c, CURLH_HEADER, 0, prev))) {
+		if (strcmp("Link", h->name) == 0 ||
+		    strcmp("link", h->name) == 0) {
+			auto value = std::string(h->value);
+			for (auto end = value.find(",");
```

Comment:
I'd also call out this parsing here is for having multiple values under the same header

Reply:
Added a note